fake-server/app.js

```diff
@@ -29,6 +33,13 @@ app.get("/get-sensors/:_id", (req, res) => {
 app.post("/curing-table", (req, res) => {
   res.send();
 });
+app.post("/delete-user", (req, res) => {
```

Comment:
We agreed that you need to send the user's id as a param eg: `/delete-user/:_id`. This should also be a [DELETE](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/DELETE) method, not a POST. Check the API issue carefully #10